tests/components/homekit_controller/conftest.py

```diff
@@ -18,9 +18,8 @@ def utcnow(request):
     """Freeze time at a known point."""
```

Comment:
I think we should rename the fixture to something that describes better what the fixture is doing.

Reply:
Good point, will do this later

Reply:
Renamed the fixture to `freeze_time_in_future`